.github/workflows/linux.yml

```diff
@@ -95,13 +97,25 @@ jobs:
       OV_TOKENIZERS_INSTALL_DIR: ${{ github.workspace }}/openvino_tokenizers/install
       OPENVINO_TOKENIZERS_REPO: ${{ github.workspace }}/openvino_tokenizers
       BUILD_DIR: ${{ github.workspace }}/openvino_tokenizers/build
+      MANIFEST_PATH: ${{ github.workspace }}/manifest.yml
 
     steps:
       - name: Clone Openvino tokenizers
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           path: ${{ env.OPENVINO_TOKENIZERS_REPO }}
 
+      - name: Generate product manifest
+        id: create_manifest
+        uses: openvinotoolkit/openvino/.github/actions/create_manifest@master
+        with:
+          repos: |
+            ${{ env.OPENVINO_TOKENIZERS_REPO }}
+          product_type: ${{ env.PRODUCT_TYPE }}
+          target_arch: 'x86_64'
+          build_type: 'release'
```

Comment:
Can we use build_type from strategy here?

Reply:
With additional hacks to convert uppercase -> lowercase in the first letter - yes